src/test/datascience/notebook/executionService.vscode.test.ts

```diff
@@ -123,6 +123,14 @@ suite('DataScience - VSCode Notebook - (Execution) (slow)', function () {
 
         await Promise.all([runCell(cell), waitForTextOutput(cell, '123412341234')]);
     });
+    test('Test __vsc_ipynb_file__ defined in cell using VSCode Kernel', async () => {
+        const uri = vscodeNotebook.activeNotebookEditor?.document.uri;
+        if (uri && uri.scheme === 'file') {
+            await insertCodeCell('print(__vsc_ipynb_file__)', { index: 0 });
```

Comment:
Nevermind about the remote stuff. I found the problem. Updated your branch with what I hope is the fix.